test/template-tests/README.md

```diff
@@ -0,0 +1,58 @@
+﻿### Running Tests
+
+Test can be run directly in PowerShell, or run from the command line using a wrapper script.
+
+You can run the full suite of tests by using Test-AzureRMTemplate.cmd (on Windows) or Test-AzureRMTemplate.sh (on Linux), and passing in the path to a template.
+
+This will run the full suite of applicable tests on your template.  To run a specific group of tests, use:
+
+
+    Test-AzureRMTemplate -TemplatePath $thePathToYourTemplate -Test deploymentTemplate 
+    # This will run deployment template tests on all appropriate files

```

Comment:
can you add what the other options are for "groups"?  e.g. deploymentTemplate, createUIDef, all